### DISCUSSION TITLE
Use [time] & [weather] signs to display current time & weather

### DISCUSSION THREAD

**Discussion by molor** [EssentialsX/Essentials#4126] [Ideas and Feature Suggestions] [answered]
Hello. Please add ability to use these signs without setting the weather/time.

e.g. right click on these signs (if lines 2-4 are empty) will display current time/weather in the chat, similar to how currently [balance] sign work. Currently, [time/weather] signs with empty 2-4 lines can't be created :c

**Comment by JRoy** [comment, marked as answer]
Added in #4131
